server/chat-commands/info.js

```diff
@@ -2401,16 +2401,17 @@ const commands = {
 	'!code': true,
 	code(target, room, user) {
 		if (!target) return this.parse('/help code');
-		if (!this.canTalk()) return;
-		if (!this.runBroadcast(true, '!code')) return;
 		if (target.length >= 8192) return this.errorReply("Your code must be under 8192 characters long!");
 
 		const params = target.split('\n');
 		if (!params[0]) params.unshift();
 		if (!params[params.length - 1]) params.pop();
-		if (params.length === 1 && params[0].length < 80 && !target.includes('```') && this.message.startsWith('/')) {
-			return `\`\`\`${params[0]}\`\`\``;
+		if (params.length === 1 && params[0].length < 80 && !params[0].includes('```') && this.willBroadcast) {
```

Comment:
`this.willBroadcast` is always `false` here, isn't it?

Reply:
Also changing `target` to `params[0]` is pointless.

Reply:
`this.willBroadcast` is `false` with `/code`, but `true` with `!code`. `params[0]` may have fewer characters if there was a leading or trailing newline.

Reply:
I don't believe you.

Reply:
![image](https://user-images.githubusercontent.com/551184/71303768-b72cfc00-2400-11ea-9aa0-3d927f40c3a5.png)

Reply:
Oh, you introduced it in the PR. It shouldn't be a getter; getters are mostly banned.